### DISCUSSION TITLE
Very difficult for untrained users

### DISCUSSION THREAD

**Discussion by Sok46** [OvertureMaps/data#63] [Discussion]
I feel very stupid looking at the data structure. I'm a simple geoanalyst who knows Qgis and Python. Understanding circuits and code is quite difficult for an untrained user. It would be great if there was a video tutorial on how to get the data correctly

**Comment by yharby** [comment]
@Sok46 
I am working on developing a tool specifically for your need _ still in heavy development _ but you can try it and give me a feedback/bugs which part is hard to da/understand , to make it easier 

https://github.com/Youssef-Harby/OvertureMapsDownloader

**Reply by mtravis** [reply]
@Sok46 If you know python then you should really check out the duckdb library. 

There are some good notes here on how to use it to read Overture Data: https://til.simonwillison.net/overture-maps/overture-maps-parquet

Also worth checking out https://beta.source.coop/cholmes/overture, where you can get the geoparquet data directly and open in QGIS

**Reply by Sok46** [reply]
Thank you, the example with DuckDB helped, since you don’t need to create an account there. It doesn’t work through Azur, because an active account is required there, and since I’m from Russia, I can’t create an account there

**Reply by mtravis** [reply]
You don't need an AWS account to access the data. See https://github.com/OvertureMaps/data/#5-download-the-parquet-files

**Comment by hkeze** [comment]
@mtravis I have to say that this link https://beta.source.coop/cholmes/overture was the best solution for me. Although my QGIS doesn't seem to being able to handle it.

**Reply by mtravis** [reply]
Are you on Windows. Grab the latest version of QGIS (3.32) and it should work work with that. 

If you're on Linux or Mac then you'll need to install [conda](https://docs.conda.io/en/latest/)  create an enviroment and run this command `conda install qgis libgdal-arrow-parquet`, and then just type `qgis` in the terminal

**Reply by hkeze** [reply]
Hey, I am on Windows and running the latest version of QGIS. 

I am also able to open the parquet file, but whenever I am trying to do anything, like opening the attribute table or trying to cut the layer by overlaying it with another one QGIS crashes completely.

**Reply by mtravis** [reply]
Exactly which parquet file are you trying to open?  Try exporting the file as a Geopackage and then perform any spatial operations or otherwise on that in QGIS instead.
